flavors/squid_nlb_central/updatewhitelist.sh

```diff
@@ -22,23 +22,23 @@ fi
 
```

Comment:
If this script runs as the `ubuntu` user, then I think these `rsync` commands need to be run with `sudo` too